database/create_user.sql

```diff
@@ -0,0 +1,3 @@
+CREATE USER 'sqluser'@'%' IDENTIFIED WITH mysql_native_password BY 'password';
+GRANT ALL PRIVILEGES ON *.* TO 'sqluser'@'%';
```

Comment:
@shirmoran Two suggestions.
- The grant command is similar to super user. I would say give all privileges on those two tables (relevant to collector) only. Otherwise we run the risk of deleting the database which we may not want.
- Rename the user to "collectoruser"